src/frame/sdk/DocumentApi.ts

```diff
@@ -16,50 +17,43 @@ import * as Utils from "../../lib/Utils";
 import {documentToByteParts, combineDocumentParts, encryptedDocumentToBase64} from "../FrameUtils";
 import SDKError from "../../lib/SDKError";
 import {UserAndGroupTypes, ErrorCodes} from "../../Constants";
+import PolicyEndpoints, {UserOrGroupWithKey} from "../endpoints/PolicyApiEndpoints";
 
 /**
- * Get a list of user and group keys for the provided users and then also add in the current user to the list of user keys
- * @param {string[]} userGrants  List of users to get public keys for
- * @param {string[]} groupGrants List of groups to get public keys for
- * @param {boolean}  grantToAuthor If the logged in user should be included in the list of users to encrypt to.
+ * Create the error strings for missing users/groups.
+ * @param userKeys The response from UserList
+ * @param groupKeys The response from GroupList
+ * @param invalidPolicyUsersAndGroups The invalid groups/users from policy application.
+ * @param userGrants The list of all the user grants.
+ * @param groupGrants The list of all the group grants.
```

Comment:
No need to change, but in the future, feel free to just nuke all these `@param` lines in the comment blocks. I've usually reverted to having the doc comment have a description of what the function does, but not commenting each parameter since our penchant for good names and TS type info really gives you enough info.

Reply:
I'll delete it now since it's new.